index.bs

```diff
@@ -134,6 +134,11 @@ handle additional, application-specific use cases.
     DocumentFragment sanitize(SanitizerInput input);
     DOMString sanitizeToString(SanitizerInput input);
 
+    static readonly attribute Sanitizer default;
+    static readonly attribute Sanitizer nofetch;
+    static readonly attribute Sanitizer nonavigate;
+    static readonly attribute Sanitizer richtext;
```

Comment:
I think it would be great if we could adopt a list of reasonable HTML elements from "somewhere else", which would allow us to inherit a widely-used set of HTML elements and thus satisfy very common use cases.
https://commonmark.org/ comes to mind, it should be somewhat easy to extract the list of html elements from their spec and provide a Sanitizer.CONFIG_COMMONMARK.